src/meshcat/visualizer.py

```diff
@@ -65,6 +65,19 @@ def get_scene(self):
         # we receive the HTML as utf-8-encoded, so decode here
         return self.zmq_socket.recv().decode('utf-8')
 
+    def get_image(self, save_path=""):
+        import io
+        from PIL import Image
```

Comment:
Here too, let's move the imports to the top of the file. We'll also need to add `Pillow` (with some appropriate version bound) to the requirements list in `setup.py`

Reply:
Ok ! Let's go for `"pillow >= 7.0.0"`, it shouldn't be too conservative